bedrock/foundation/templates/foundation/trademarks/list.html

```diff
@@ -19,9 +19,15 @@ <h1 class="page-title">{{ _('List of Mozilla Trademarks') }}</h1>
       <th>{{ _('Firefox logo') }}&reg;</th>
     </tr>
     <tr>
+      <td>
+        {{ high_res_img('trademarks/hello-logo-wordmark.png', { 'alt': 'Firefox Hello logo' }) }}
```

Comment:
I don't believe it is localized (but @flodolo can confirm). Either way, existing strings are already wrapped on this page so we should still wrap new ones.